remotion/compositions/templates/talk/Talks.composition.tsx

```diff
@@ -33,16 +36,16 @@ export const TalksComposition: React.FC = () => {
 			/>
 			<Composition
 				component={TalkBranded}
-				width={1200}
-				height={700}
+				width={1280}
+				height={720}
 				id="TalkBranded"
 				fps={30}
 				durationInFrames={140}
 				defaultProps={{
 					title: 'Certification “Google Cloud Architect”',
-					startingDate: new Date(2023, 3, 18, 13),
-					endingDate: new Date(2023, 4, 23, 13, 45),
-					reccuringDay: 'mardi',
+					startingDate: startingDate,
+					endingDate: endingDate,
```

Comment:
On peut simplifier ici

```suggestion
					startingDate,
					endingDate,
```